### PR TITLE
feat: Add error handler and user can choose text storage option

### PR DESCRIPTION
**Pull request by chickennight**
Error Handler Added: error will not end node.js
UX upgrade: in txt file output case, user can choose data storage method

### CODE REVIEW

File: GOOD-I-DEER/nodes/face-vectorization-node/face-vectorization.html [hunk 1]
```diff
@@ -41,8 +42,8 @@
     label: function () {
       return this.name || "Good Face Vectorization";
     },
-    inputLabels: ["Single Image"],
-    outputLabels: ["Vector"],
+    inputLabels: ["Single Face Image"],
+    outputLabels: ["FAce Vector Array"],
```
Comment (leecr1215) [inline GOOD-I-DEER/nodes/face-vectorization-node/face-vectorization.html]: Did you mean ['Face Vector Array'] ?

File: GOOD-I-DEER/nodes/face-vectorization-node/face-vectorization.js [hunk 2]
```diff
@@ -15,72 +15,99 @@ module.exports = function (RED) {
...
     async function image_vectorization(inputData) {
-      const model = await onnx.InferenceSession.create(modelPath);
+      try {
+        const model = await onnx.InferenceSession.create(modelPath);
 
-      const inputName = model.inputNames[0];
+        const inputName = model.inputNames[0];
 
-      const inputDims = [1, 3, 224, 224];
+        const inputDims = [1, 3, 224, 224];
```
Comment (ramaking) [inline GOOD-I-DEER/nodes/face-vectorization-node/face-vectorization.js:61]: Loading the model and defining settings only needs to be done once! 😘

File: GOOD-I-DEER/nodes/face-vectorization-node/face-vectorization.js [hunk 3]
```diff
@@ -15,72 +15,99 @@ module.exports = function (RED) {
...
+          }
         }
-
-        fs.writeFileSync(filePath, textData, "utf-8");
+      } catch (error) {
+        node.error("An error occurred: " + error.message.split("\n")[0]);
       }
     });
 
     async function image_vectorization(inputData) {
```
Comment (Gitgloo) [inline GOOD-I-DEER/nodes/face-vectorization-node/face-vectorization.js]: how about use Camel Case?

File: GOOD-I-DEER/nodes/face-vectorization-node/face-vectorization.js (hunk 2 above)
Comment (chickennight) [reply GOOD-I-DEER/nodes/face-vectorization-node/face-vectorization.js:61]: This is evidence of an attempt to allow users to choose from a variety of models. I plan to leave it for expandability.

File: GOOD-I-DEER/nodes/face-vectorization-node/face-vectorization.html (hunk 1 above)
Comment (chickennight) [reply GOOD-I-DEER/nodes/face-vectorization-node/face-vectorization.html]: fixed, thanks!

File: GOOD-I-DEER/nodes/face-vectorization-node/face-vectorization.js (hunk 3 above)
Comment (chickennight) [reply GOOD-I-DEER/nodes/face-vectorization-node/face-vectorization.js]: all form change to Camel Case thanks

**Review by hy06ix — APPROVED** [review]

**Review by KimJaea — APPROVED** [review]

**Pull request merged**